packages/govuk-react/package.json

```diff
@@ -63,7 +63,7 @@
   },
   "peerDependencies": {
     "react": ">=16.2.0",
-    "styled-components": "^4.1.3"
+    "styled-components": ">=5.1"
```

Comment:
huh how did I miss that

Reply:
dunno - I spotted it whilst one of my many attempted builds was happening :grin: